client/js/views/GameView.vue

```diff
@@ -1253,30 +1249,68 @@ export default {
 .history-container {
   grid-area: history;
 }
+
 #field-right {
   height: 100%;
+  padding-top: 0.5rem;
 
   #history,
   #card-preview {
     margin: 0 auto;
-    padding: 5px;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    border: 1px solid #fff;
+    padding: 0.5rem;
     width: 80%;
     max-width: 80%;
     height: 100%;
+    border-radius: 20px;
+    box-shadow: 0px 0px 10px #111111;
   }
   #history {
-    background-color: rgba(0, 0, 0, 0.46);
+    background-color: rgba(241, 200, 160, 0.65);
+    color: #111111;
     & #history-logs {
-      height: 80%;
       overflow: auto;
+      height: 85%;
+      font-size: 0.75em;
+      letter-spacing: 0.25px;
+      font-family: 'Libre Baskerville', serif;
```

Comment:
@itsalaidbacklife same with the fonts

Reply:
Yeah the fonts are actually more troublesome than the colors as far as where to put them because we're not being very semantically consistent overall with them. I'll write up a font reorganization ticket

Reply:
We need logical fallbacks for all these fonts, something like a css variable in a config:
```
--font-family-primary: 'Libre Baskerville', Georgia,Times,Times New Roman,serif;
```

and then later on, you can use it like this:
```
font-family: var(--font-family-primary);
```

Reply:
Added font fall backs to these in the latest commit.